z/btree.go

```diff
@@ -51,22 +52,44 @@ func (t *Tree) initRootNode() {
 	t.Set(absoluteMax, 0)
 }
 
-// NewTree returns a memory mapped B+ tree with given filename.
+// NewTree returns an in-memory B+ tree.
 func NewTree() *Tree {
-	t := &Tree{}
+	t := &Tree{buffer: NewBuffer(minSize, "tree")}
```

Comment:
The NewBuffer by default uses calloc. The current implementation was using go memory. With this change, we will use calloc'ed memory instead of go memory for the btree.

Reply:
Asked Manish, he said Calloc should be fine.